local-cli/desktop/genDesktop.js

```diff
@@ -15,7 +15,7 @@ var generate = require('../generate/generate');
 const { exec } = require('child_process');
 
 function applyDesktopPlatformPatch() {
-  exec('pwd && patch --verbose -d ./node_modules/metro/src -i ../../react-native/add-desktop-platform.patch', (err, stdout, stderr) => {
+  exec('git apply ./node_modules/react-native/patches/metro-config+0.48.3.patch', (err, stdout, stderr) => {
```

Comment:
A more cross-platform way to apply patches.  Needs error handling when patch is already applied, though.